kubernetes/apps/monitoring/kube-prometheus-stack/app/helmrelease.yaml

```diff
@@ -195,9 +195,9 @@ spec:
         # externalLabels:
         #   cluster: home # must match with thanos value `.ruler.clusterName`
         thanos:
-          image: quay.io/thanos/thanos:v0.30.1
+          image: quay.io/thanos/thanos:v0.30.2
           # renovate: datasource=docker depName=quay.io/thanos/thanos
-          version: "v0.30.1"
+          version: "v0.30.2"
```

Comment:
Figured out you don't need like 199/200.  See my https://github.com/szinn/k8s-homelab/commit/1ec1476b6738e5a34cfb090ea37224bf77dda37b

Also fixes the problem with getting a different code from ecr

Reply:
My only concern with that is if bitnami releases a chart update that breaks using the official image, or if an official image update breaks the chart.

Reply:
True - however the quay.io version is being used in kps so should match. Bitnami is taking their sweet time publishing the updated image even though they refer to it in their chart to ecr.aws. Guess they only publish to docker.io perhaps?

Reply:
Their images on ecr tend to lag a day or two. Dockerhub doesn't taste limit Bitnami images, so using their dockerhub images isn't bad.

Reply:
I'll go ahead with your changes and I'll try to keep a mental note on future updates.

Reply:
Ah, goot to know about the rate-limiting thing.